src/main/clojure/clara/rules.cljc

```diff
@@ -332,6 +332,12 @@
       * :forms-per-eval - The maximum number of expressions that will be evaluated per call to eval.
         Larger batch sizes should see better performance compared to smaller batch sizes. (Only applicable to Clojure)
         Defaults to 5000, see clara.rules.compiler/forms-per-eval-default for more information.
+      * :omit-compile-ctx - When false Clara, in Clojure, retains additional information to improve error messages during
+        session deserialization at the cost of additional memory use.
+        By default this information is retained till the session is initially compiled and then will be discarded,
+        in the event that the Session will be serialized and deserialized (Durability) this information could be useful
```

Comment:
I think this can just say "serialized", the "and deserialized" is not necessary in cases where you don't really need to distinguish the 2 sides of serialization.

Also, instead of "(Durability)", how about "(ie. via Clara's durability support)".